api/api.js

```diff
@@ -16,6 +18,7 @@ async function request(path, { token, ...options }) {
   if (response.ok) {
     return response.json();
   } else {
+    logger.error(`Fetching ${url} failed`, response);
```

Comment:
Not 100% sure, but the response object _could_ contain circular references that will cause this to throw `TypeError: Converting circular structure to JSON`. Might be worth just logging the status code maybe?

Reply:
righto